.tools/validate.go

```diff
@@ -16,6 +16,12 @@ import (
 var DefaultRules = []ValidateRule{
 	func(c CommitEntry) (vr ValidateResult) {
 		vr.CommitEntry = c
+		if len(strings.Split(c.Parent, " ")) > 1 {
```

Comment:
Instead of splitting at consume-time, it seems cleaner to make `CommitEntry.Parent` a `CommitEntry.Parents` array and parse at commit-lookup time.